Model/UserManager.php

```diff
@@ -22,8 +22,8 @@
  */
 abstract class UserManager implements UserManagerInterface
 {
-    private $passwordUpdater;
-    private $canonicalFieldsUpdater;
+    protected $passwordUpdater;
+    protected $canonicalFieldsUpdater;
```

Comment:
Well, inside the getter method you could trigger a deprecation making people aware of the fact that the method will be removed with the next major release. That's not possible when dealing with access to properties.